src/fetch/FetchExample.re

```diff
@@ -0,0 +1,87 @@
+type dog = string;
+
+type arrayDogs = array(dog);
```

Comment:
I still don't understand the purpose of this. Why not just use `array(dog)` directly?

Reply:
I will update that.